openedx/core/djangoapps/oauth_dispatch/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""
+Root of the oauth_dispatch app.
+
+This app dispatches oauth request processing to django-oauth2-provider (DOP) or
+django-oauth-toolkit (DOT).
+
+"""
+
+default_app_config = 'openedx.core.djangoapps.oauth_dispatch.OAuthDispatchAppConfig'
```

Comment:
Why was this change needed?  Also, shouldn't the path include 'apps'?

Reply:
Yes, it would need apps.  This wasn't needed.  Removed.

Also, I had misremembered something.  I thought that having `default_app_config` was a best practice, but that was my bad.  The best practice I was remembering was having the AppConfig which is referred to in `INSTALLED_APPS`.  So, this was pointless and broken. :)